zzzz_modular_occulus/code/game/machinery/dorms_elevator.dm

```diff
@@ -4,9 +4,17 @@
 
 /obj/machinery/button/remote/elevator_panel/proc/despawn_passenger(var/mob/living/user)
 	var/mob/living/passenger = user
+	//var/mob/living/carbon/human/passengerplayer = user
```

Comment:
Ignore this. I was testing another method that didn't quite work and forgot to delete it. I'll delete it and info comment all this code properly once I get around to adding the next update for the elevator